src/components/Mermaid.astro

```diff
@@ -0,0 +1,6 @@
+---
+const {code} = Astro.props;
+---
+<pre class="mermaid my-8 rounded-lg border border-outline-opaque bg-surface-secondary">
```

Comment:
Change: Center the mermaid diagrams. 
You can do this by wrapping the `pre` element with a `center` element.

Before:
<img width="1109" alt="image" src="https://github.com/user-attachments/assets/dcf4b36e-7b7f-4615-9c66-6d384dd0bcc5">

After:
<img width="1109" alt="image" src="https://github.com/user-attachments/assets/c3aba999-945f-475b-9fb3-d7697af9fff9">